types/bingmaps/Autosuggest.d.ts

```diff
@@ -0,0 +1,138 @@
+﻿/*
```

Comment:
We can't have differing licenses in this project due to automation. This is the same as the root MIT license, which is redistributed automatically with each `@types/` package, so there's no need to place it in source